AK/JsonValue.h

```diff
@@ -91,6 +91,22 @@ class JsonValue {
     u32 to_u32(u32 default_value = 0) const { return to_number<u32>(default_value); }
     u64 to_u64(u64 default_value = 0) const { return to_number<u64>(default_value); }
 
+    String to_bcd(u32 default_value = 0) const
```

Comment:
I'm not sure about this specifically being in `AK::JsonValue`, considering it's only used once in `lsusb` and will probably only _ever_ be used there..